<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_28">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_24">
     <item>
      <widget class="QPushButton" name="ug_button">
       <property name="text">
        <string>Ucitaj Gradove</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Broj Posetilaca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bp_lineedit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="zp_button">
       <property name="text">
        <string>Zapocni posete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_25">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gradovi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lw_cities"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_27">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Prikupljeni artikli</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lw_articles"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Predjeni put:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pp_lineedit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
